.github/workflows/lint-test.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
 
     - name: Run artifacthub.io changelog check
       run: |
-        for chartyaml in $(git diff --name-only origin/main | awk -F/ '/^charts\// { printf "%s/%s/%s\n",$1,$2,"Chart.yaml" }' | sort -u); do
+        for chartyaml in $(git diff --name-only origin/main -- ':(exclude)*/ci/linter-values.yaml' | awk -F/ '/^charts\// { printf "%s/%s/%s\n",$1,$2,"Chart.yaml" }' | sort -u); do
```

Comment:
I did this because linter-values only changes in a chart shouldn't need a change log